package.json

```diff
@@ -87,6 +83,7 @@
   },
   "license": "MIT",
   "scripts": {
+    "preinstall": "npm install node-pre-gyp node-pre-gyp-github",
```

Comment:
It should be harmless but I doubt you need to have `node-pre-gyp-github` in the `preinstall` deps. I presume it can just be a normal dep since it only needs to be used by developers when posting new binaries (not when installing), if I'm understanding its purpose correctly.